src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -621,6 +621,21 @@ public async Task Timeout_SetTo30AndGetResponseQuickly_Success()
             }
         }
 
+        [Theory]
+        [InlineData(HttpCompletionOption.ResponseContentRead)]
+        [InlineData(HttpCompletionOption.ResponseHeadersRead)]
+        public async Task Timeout_TimeoutTriggeredOnBufferedResponse_ThrowsNestedTimeoutException(HttpCompletionOption completionOption)
```

Comment:
Don't we already have tests for HttpClient.Timeout?  Can we just augment those?

Reply:
It seems Timeout_TooShort_AllPendingOperationsCanceled does almost the same, so I merged the check into that and removed this one.